tests/test-update-refspec-checkpoint.c

```diff
@@ -0,0 +1,238 @@
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
+ *
+ * Copyright © 2016 Kinvolk GmbH
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * Authors:
+ *  - Krzesimir Nowak <krzesimir@kinvolk.io>
+ */
+
+#include "misc-utils.h"
+#include "spawn-utils.h"
+#include "eos-test-utils.h"
+
+#include <gio/gio.h>
+#include <locale.h>
+
+const gchar *next_ref = "REFv2";
+const gchar *next_refspec = "REMOTE:REFv2";
+
+static void
+following_refspec_files (guint         commit,
```

Comment:
This function could do with a brief comment saying what it does too. The function name is not very obvious to me.